frontend/lib/Views/AdminPage/utils.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+import '../enpoint_list_view.dart';
+import 'groups_view.dart';
+
+MaterialStateProperty<EdgeInsetsGeometry?> buttonPadding =
+    MaterialStateProperty.all(const EdgeInsets.all(20));
+
+ButtonStyle proceedButtonStyle = ButtonStyle(
+  side: MaterialStateProperty.all(
+    const BorderSide(width: 1, color: Colors.grey),
+  ),
+  padding: buttonPadding,
+  backgroundColor: MaterialStateProperty.all(floatingButtonColor),
+  shape: MaterialStateProperty.all(
+    RoundedRectangleBorder(borderRadius: basicBorderRadius),
+  ),
+  alignment: Alignment.centerLeft,
+);
+
+TextStyle proceedButtonTextStyle =
+    const TextStyle(fontFamily: 'SofiaSans', fontSize: 16, color: Colors.white);
+
+Widget cancelButton(BuildContext context) => OutlinedButton(
+      onPressed: () {
+        Navigator.of(context).pop();
+      },
```

Comment:
Czy nie lepiej by było, żeby `cancelButton` przyjmował funkcję `onCancel` i ją wywoływał przy wciśnięciu? Dzięki temu nasz button nie będzie miał pojęcią o kontekście i mógłby być reużywalny w innych przypadkach :thinking:

Reply:
on juz jest reużywalny, bo kazdy cancel button robi to samo - zamyka modala i pomija akcje

Reply:
Zgadzam się, ale mógłby być również re-używalny w innych widokach, nie tylko w modalu, a nie wiadomo, czy przy cancelowaniu nie chcemy jeszcze wykonać jakiejś akcji